packages/expo/src/removed.ts

```diff
@@ -698,6 +710,17 @@ Object.defineProperties(module.exports, {
     },
   },
   /** @deprecated */
+  Pedometer: {
```

Comment:
usually best to keep changes like this to separate commits, same for reordering lines from removed.web.ts below.